Changed dpop specs for ASs (#1144)
italia/eidas-it-wallet-docs: docs/en/e-service-pdnd.rst

    - R1, R2, R4, R5

.. note::

>     In these specifications, the ``REST_JWS_2021_POP`` security pattern is implemented by default in accordance with :rfc:`9449`.

Reviewer: ```suggestion
    In the current specifications, the ``REST_JWS_2021_POP`` security pattern is implemented in accordance with :rfc:`9449`.
```

>     Alternatively, the proof of possession MAY be attested by the ``TrackingEvidence`` JWT (as detailed below). However, while the ``TrackingEvidence`` is defined in ``AUDIT_REST_02`` to provide additional tracked data, in this context, it acts as proof of possession of the Voucher. Such implementation choices will be referred to as ``POP_DPoP`` and ``POP_TPoP``, respectively.

Reviewer: ```suggestion
    Alternatively, the proof of possession MAY be attested using the ``TrackingEvidence`` JWT to provide additional data. In this context, ``TrackingEvidence`` JWT acts as a proof of possession of the PDND Voucher. Such implementation choices will be referred to as ``POP_DPoP`` and ``POP_TPoP``, respectively.
```

>     Alternatively, the proof of possession COULD be attested by the ``TrackingEvidence`` JWT (as detailed below). However, while the ``TrackingEvidence`` is defined in ``AUDIT_REST_02`` to provide additional tracked data, in this context, it acts as proof of possession of the Voucher. Such implementation choices will be referred to as ``POP_DPoP`` and ``POP_TPoP``, respectively.

Reviewer: ```suggestion
    Alternatively, the proof of possession MAY be attested by the ``TrackingEvidence`` JWT (as detailed below). However, while the ``TrackingEvidence`` is defined in ``AUDIT_REST_02`` to provide additional tracked data, in this context, it acts as proof of possession of the Voucher. Such implementation choices will be referred to as ``POP_DPoP`` and ``POP_TPoP``, respectively.
```
